lang_python/parsing/Parser_python.mly

```diff
@@ -156,6 +156,7 @@ let mk_str ii =
 
 %token <AST_python.tok> 
   EQ "="             (* = *)
+  COLONEQ ":="   (* := *)
```

Comment:
Note that the ability to assign back a string to a token is a special feature of menhir (it's not in ocamlyacc)
which then allow you in the rule to simply use ":=" instead of the more obscure COLONEQ.
Again, just FYI. I don't always use this feature, but for some tokens that are very often used ("(", "=", etc.) I tend
to use it because it makes the grammar rule more readable.